cns/restserver/ipam.go

```diff
@@ -190,6 +190,38 @@ func (service *HTTPRestService) GetPodIPConfigState() map[string]cns.IPConfigura
 	return service.PodIPConfigState
 }
 
+func (service *HTTPRestService) GetHTTPRestDataHandler(w http.ResponseWriter, r *http.Request) {
+	var (
+		resp          GetHTTPServiceDataResponse
+		returnMessage string
+		err           error
+	)
+
+	defer func() {
+		if err != nil {
```

Comment:
i just made a similar comment in another file (apologies for not catching and discussing earlier), but this will never be nil in this line. function can be made simpler